src/algorithms/groundstate/gradient_grassmann.jl

```diff
@@ -54,16 +54,16 @@ function find_groundstate(ψ::S, H, alg::GradientGrassmann,
         @warn "This is not fully supported - split the mps up in a sum of mps's and optimize seperately"
     normalize!(ψ)
 
-    #optimtest(GrassmannMPS.fg,(ψ,envs);alpha=-0.01:0.001:0.01,retract=GrassmannMPS.retract,inner=GrassmannMPS.inner)
-    x, _, _, _, normgradhistory = optimize(GrassmannMPS.fg,
-                                           GrassmannMPS.ManifoldPoint(ψ, envs),
+    fg(x) = GrassmannMPS.fg(x, H, envs)
+    x, _, _, _, normgradhistory = optimize(fg, ψ,
                                            alg.method;
                                            (transport!)=GrassmannMPS.transport!,
                                            retract=GrassmannMPS.retract,
                                            inner=GrassmannMPS.inner,
-                                           (scale!)=GrassmannMPS.scale!,
-                                           (add!)=GrassmannMPS.add!,
+                                           (scale!)=VectorInterface.scale!,
```

Comment:
Do you need these parentheses because it is otherwise interpreted as `!=`. I guess I used spaces around `=` even for keyword arguments before we started to rely on JuliaFormatter.jl. So this is a good reason to never have keyword arguments that end with exclamation marks?

Reply:
Indeed, the formatter brings them together, and `!=` has precedence. Interestingly, I just checked, and the implicit keyword syntax does actually work: `f(; VectorInterface.scale!) === f(; (scale!)=VectorInterface.scale!)` so I might still change that